test/freenet/client/filter/M3UFilterTest.java

```diff
@@ -18,9 +18,10 @@
     };
 
     private static final String BASE_URI_PROTOCOL = "http";
+    private static final String HOST_PORT = "localhost:8888";
     private static final String BASE_URI_CONTENT = "localhost:8888";
     private static final String BASE_KEY = "USK@0I8gctpUE32CM0iQhXaYpCMvtPPGfT4pjXm01oid5Zc,3dAcn4fX2LyxO6uCnWFTx-2HKZ89uruurcKwLSCxbZ4,AQACAAE/FakeM3UHostingFreesite/23/";
-    private static final String BASE_URI = BASE_URI_PROTOCOL+"://"+BASE_URI_CONTENT+'/'+BASE_KEY;
+    private static final String BASE_URI = '/'+BASE_KEY;
```

Comment:
With this change, `BASE_URI_PROTOCOL` and `BASE_URI_CONTENT` are now unused; please remove them.

Reply:
Also, spaces around operators, please! :)

Reply:
done :-)